ompi/mpi/fortran/use-mpi-f08/mpi-f-interfaces-bind.h

```diff
@@ -810,7 +811,8 @@ subroutine ompi_iallgather_f(sendbuf,sendcount,sendtype,recvbuf, &
                             recvcount,recvtype,comm,request,ierror) &
    BIND(C, name="ompi_iallgather_f")
    implicit none
-   OMPI_FORTRAN_IGNORE_TKR_TYPE, INTENT(IN) :: sendbuf, recvbuf
+   OMPI_FORTRAN_IGNORE_TKR_TYPE, INTENT(IN), ASYNCHRONOUS :: sendbuf, recvbuf
```

Comment:
`sendbuf` and `recvbuf` cannot be defined twice

Reply:
I saw, that too.  I'm also getting a compile error that I'm puzzling over:

```
ialltoallw_f08.F90:35:53:

    call ompi_ialltoallw_f(sendbuf,sendcounts,sdispls,sendtypes,&
                                                     1
Error: Type mismatch in argument ‘sendtypes’ at (1); passed TYPE(mpi_datatype) to INTEGER(4)
```

Reply:
Oops -- I originally pasted in the wrong compile error.  This is the right one:

```
  PPFC     ialltoallw_f08.lo
ialltoallw_f08.F90:35:53:

    call ompi_ialltoallw_f(sendbuf,sendcounts,sdispls,sendtypes(1)%MPI_VAL,&
                                                     1
Error: Rank mismatch in argument ‘sendtypes’ at (1) (rank-1 and scalar)
```

Reply:
Ah, I see the problem.  Directed comment coming up...